pages/docs/administration/getting-started/configuration/app-configuration.mdx

```diff
@@ -0,0 +1,138 @@
+---
+priority: 100100
+---
+
+import DocsWrapper from '@/src/docs/DocsWrapper'
+
+# App Configuration
+
+export default ({ children }) => <DocsWrapper>{children}</DocsWrapper>
+
+## Note
+
+Below listed configurations can be changed during dockerbuild.
+
+- default application configuration is applied from config/default.js.
+- production.js overwrites mongo configuration.
+- production.js overwrites minio configuration.
+- production.js overwrites registry configurration.
+- build.environment: openshift //must be set for OpenShift
+- openshift.[ namespace | appPublicRoute ] //must be set for OpenShift
+- smtp.enabled: true //must be set, else Bailo does not start the s2i build
+- logging.file.enabled: false //set true to debug Bailo
+
+## production.js
+
+production.js can be overwritten during HELM deployment. See helm/bailo/templates/bailo/bailo.configmap.yaml
+
+## Example **default.js**
+
+```
```

Comment:
This isn't an example, this _is_ the default configuration.  It shouldn't be included here, as it changes often.  Instead, you should link / direct the user to the location.

You should instead talk about `node-config` and how it overrides files (including the order of those overrides).